src/components/swimlanes/swimLanes.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  CardHeading,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Spacer,
+  Text,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+
+const SwimLanes = ({ lanesWithNumItems }) => {
+  return (
+    <>
+      {lanesWithNumItems.map((lane, key) => (
+        <Box data-testid={lane.slug} key={lane.rank} mb="var(--nypl-space-l)">
+          <Flex alignItems="baseline">
+            <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+              {lane.title}
+            </Heading>
+            <Spacer />
+            <Link
+              id={`row-see-more-${lane.slug}`}
+              type="standalone"
+              href={"https://digitalcollections.nypl.org/search/index?"}
+              aria-label="See more digitized materials"
```

Comment:
To be clear, it should be dynamic for every swim lane.

Reply:
Sorry, the ticket was unclear and I confused "digitized materials" with the title of the first swim lane. This would be good to get clarification from Lana if that text is intentional and is not meant to have the title of the swim lane. And also clarification from Clare if multiple links on the page with the same aria-label is accessible/clear to the user.

Reply:
I'll leave a note in the ticket rn